docs/src/SUMMARY.md

```diff
@@ -62,6 +62,7 @@
     - [forc deploy](./forc/commands/forc_deploy.md)
     - [forc init](./forc/commands/forc_init.md)
     - [forc json-abi](./forc/commands/forc_json-abi.md)
+    - [forc json-storage-slots](./forc/commands/forc_json-storage-slots.md)
```

Comment:
Is this command actually needed? It's an automatic output from the `build` process, same with `abi`. I would vote to remove `forc json-abi` instead of introducing more redundant commands.

Reply:
Alright, so I started by stopping the e2e test from relying on `forc json-abi` and removed the new `forc storage-slots-json` command. I agree that they seem redundant except maybe for specifying an output file. We could however add a flag `-json-abi-output-file` to `forc build` (and others) to specify where the JSON file should go, but that's not too important.

I will remove `forc json-abi` from `forc` in a subsequent PR.

Reply:
>  We could however add a flag -json-abi-output-file to forc build (and others) to specify where the JSON file should go

Yep, that looks like the better alternative to having entirely new commands.